lib/renaissance_web/controllers/auction_controller.ex

```diff
@@ -60,11 +62,28 @@ defmodule RenaissanceWeb.AuctionController do
       |> render("show.html", %{
         auction: Auctions.get!(id),
         user: Auth.current_user(conn),
+        current_price: fetch_current_prices(id),
         changeset: conn
       })
     else
       {:error, changeset} ->
         render(conn, "show.html", changeset: changeset)
     end
   end
+
+  defp fetch_current_prices(id) do
+    current_price_float =
+      (Bids.get_highest_bid_amount(id) || Auctions.get!(id).price.amount)
```

Comment:
this is duped (like you can extract it)

Reply:
Resolved with https://github.com/xray/Renaissance/commit/bd0d702157b55f30fd237005ba264712b535732e#diff-35e23b1cd34500f693ff9083435c3406R29